apps/extension/src/application/trading-copilot/subscriptions-management/components/subscriptions-list/subscription-item.tsx

```diff
@@ -63,44 +69,62 @@ export const SubscriptionItem = ({ subscription, onRemove }: Properties) => {
       <div className="flex items-center">
         <LazyImage
           src={avatarQuery.data}
-          className="size-5 rounded-full"
+          className="size-8 rounded-full bg-neutral-200"
           fallbackComponent={
-            <Icon size={20} name="PersonIcon" className="rounded-full" />
+            <div className="flex size-8 items-center justify-center rounded-full bg-neutral-200">
+              <IdrissIcon
+                size={20}
+                name="CircleUserRound"
+                className="m-auto text-neutral-700"
+              />
+            </div>
           }
         />
 
-        <p className="ml-2 flex items-center gap-1 text-sm text-neutral-700">
+        <p className="ml-2 flex items-center gap-1 text-label5 font-normal text-neutral-600">
```

Comment:
do we want to keep the `font-normal` className here?
![image](https://github.com/user-attachments/assets/a17207ce-b17b-4ea0-b222-36e995e976f7)